### ISSUE TITLE
runtime error

### ISSUE THREAD

**Issue by Bikpik** [psykad/Grimoire#7]
When trying to cast a MB, rank of spell doesn't matter, during a Light SC my Geo gets the following error:

Grimoire: Lua runtime error: grimoire/grimoire.lua.158: attempt to concatenate field '?' (a nil value)

**Comment by ryanskeldon** [comment]
I originally wrote this for SCH so it'll only detect that job's spells. It can be expanded to include GEO but unfortunately I no longer have a subscription. I'll look into adding GEO spells when I have some free time.

**Comment by ryanskeldon** [comment]
Which spell were you casting? What was the command you used in your macro? i.e. grimoire cast spell 4

**Comment by Bikpik** [comment]
I was using the macro "/console grimoire cast spell 3" but any tier wouldn't work. it happens on lightsday with a light sc, as well as darksday with a dark sc with no weather for each and on Geo
________________________________

**Comment by ryanskeldon** [comment]
I just resubbed and will be checking this out. It seems like the error is occurring because there aren't any spells defined for light or dark sc. Any suggestions on what spell should be cast? I haven't played GEO yet. If you're on Asura, look for me: Kuvoyavo.

**Comment by enfarious** [comment]
Luminohelix or Noctohelix for light and dark respectively. There are only 2 tiers, I think, so may have to modify for that.

**Comment by enfarious** [comment]
I think you'd actually only be looking at subjob spells for MBing Geo maybe, I saw SCH somewhere and jumped lol

**Comment by ryanskeldon** [comment]
@enfarious The add-on is mainly for SCH or BLM but it wouldn't take a lot of effort to extend it to other casters.

**Comment by enfarious** [comment]
Yeah I was mostly thinking about what spells are available to a GEO. I imagine that anything that can MB could be worked in with a little effort, even RUN or COR abilities
